homeassistant/components/rflink/cover.py

```diff
@@ -52,12 +55,51 @@
 )
 
 
+def entity_type_for_device_id(device_id):
+    """Return entity class for protocol of a given device_id.
+
+    Async friendly.
+    """
+    entity_type_mapping = {
+        # KlikAanKlikUit cover have the controls inverted
+        "newkaku": TYPE_INVERTED
+    }
+    protocol = device_id.split("_")[0]
+    return entity_type_mapping.get(protocol, None)
```

Comment:
`None` is the default value returned if the key is missing in the dict.

Reply:
I will change this to TYPE_STANDARD and test it